test/data/simpleMessage/SimpleMessage.java

```diff
@@ -116,9 +116,17 @@ public void run() {
         }
     }
 
+    public void method_with_double_stash_as_string_parameter("\\") {
```

Comment:
Does this code compile?

Reply:
I don't see any error in our project.  
But apparently, the gramma is incorrect.

Reply:
you are right,  the parameter should be removed.